src/pudl/metadata/resources/eia860.py

```diff
@@ -906,6 +906,75 @@
         "sources": ["eia860"],
         "etl_group": "eia860",
     },
+    "core_eia860__scd_generators_multifuel": {
+        "description": (
+            "Annually reported information about energy storage from EIA-860 Schedule 3. "
+            "Contains data on fuel-switching and the use of multiple fuels by surveyed generators."
```

Comment:
I don't think the first sentence here is relevant to this table. As far as I understand it's about generators not energy storage entities. Maybe rework the sentence to say the following: 

```suggestion
        "description": (
            "Annually reported information from EIA-860 Schedule 3 about generators that use multiple fuel "
            " sources or engage in fuel switching. " 
```